tests/modules/asset_groups/resources/utils.py

```diff
@@ -60,6 +59,79 @@ def create_simple_asset_group(flask_app_client, user, request, test_root):
     return group_guid, ags_guids[0], asset_guids[0]
 
 
+# as for above but returns uuids as a dict
```

Comment:
couple of extra utils, mostly just moved from the sighting utils where they should never have been anyway.